news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,143 @@
+
+The difficulty value of colour can now be calculated using this new encoding information. **This means that the colour difficulty of each note can be defined as the amount of information it contributes to the sequence so far**. Generally, the more information-dense a given colour sequence is, the harder it is to decode and play. We can estimate the information density of a given density by the amount of encoding chunks required to describe the pattern. Finally, we try to restrict colour difficulty to consider past notes only. Hence, sequences that are exactly the same except for future notes should have identical difficulty values, up until where they differ.
+
+### Calculating the final result
+
+Given the four skills briefly described above (stamina, rhythm and colour), their strain values are combined into one final Star Rating value. Previously done in two ways, this is now done in one ways - via the new *peaks* skill. *Peak* difficulty splits the map into sections and combines peaks of each skill in each section, aiming to catch out particularly demanding moments. The final Star Rating is the outputted result, slightly multiplied to scale better. 
+
+While this is much more mathematically complex than mentioned, you can consult the osu!taiko general rework document listed in the above sections, or dive into the code over in the osu!lazer repositor for the full details. 
+
+While previous sentiments aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/en/Ranking_Criteria/Difficulty_naming#osu!taiko), this rework has deviated away from the purpose of this, aiming to reflect true difficulty, regardless of naming. This also means that high SR values with near-fc's are no longer near 9.5*, rather can expand well into the 10* range (medal hunters rejoice!)
+
+## Changes to Performance Points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values (around 40% lower). In addition, the survey indicated that the way performance points are calculated needed to be re-evaluated. 
+
+Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance not only between difficulty increasing mods, but their reductive counterparts.
+
+Served alongside the SR rework, these changes aim to accurately reflect the new sentiments of the SR system.
+
+### Globally-applied changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which consists of both difficulty and accuracy performance points. This allows for simplicity of balancing changes, and works alongside specific multipliers found within their respective separate difficulty values. 
+
+- The HD multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty PP, rather than globally. 
```

Comment:
```suggestion
- The HD multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty PP, rather than globally.
```